Frontend/ui-customer/src/MerchantConfigs/DefaultConfig.purs

```diff
@@ -128,4 +128,5 @@ config =
       { notify : false 
       , autoGeneratedText : ""
       }
+  , hideContactSupport : false
```

Comment:
change the name to `enableContactSupport` and add the default value in commonConfig for other merchants